src/js/components/modals/InstallPackageModal.js

```diff
@@ -296,6 +328,36 @@ class InstallPackageModal
     );
   }
 
+  renderBetaTermsTabView() {
+    const { title, description } = this.state.betaOptInProperties;
+
+    return (
+      <div>
+        <div className="modal-body">
+          <div className="horizontal-center">
+            <h2>{title}</h2>
+            <p className="text-align-center">
+              {description}
+            </p>
+          </div>
+        </div>
+        <div className="modal-footer">
+          <div className="button-collection button-collection-align-horizontal-center flush-bottom">
+            <button className="button" onClick={this.props.onClose}>
+              Cancel
+            </button>
+            <button
+              className="button button-success"
+              onClick={this.handleAcceptBetaTerms}
+            >
+              Accept
```

Comment:
If feels strange to me having some internationalized strings in this file and some that are not. I vote for consistency here and I also vote for not internationalizing this file now since it is tagged for a heavy refactor.